cms-adapter/src/Rsbc.Dmf.CaseManagement/CaseManager.cs

```diff
@@ -3094,7 +3100,8 @@ private int TranslateDocumentStatusCode(string documentStatusCode)
             }
             else
             {
-                return 2;
+                // default value is OTHER 
+                return 100000004;
```

Comment:
It would be better to use the dictionary declared above and reference item "OTHR" in that dictionary